paasta_tools/kubernetes/bin/paasta_secrets_sync.py

```diff
@@ -38,10 +41,23 @@
 from paasta_tools.paasta_service_config_loader import PaastaServiceConfigLoader
 from paasta_tools.secret_tools import get_secret_provider
 from paasta_tools.utils import DEFAULT_SOA_DIR
+from paasta_tools.utils import get_service_instance_list
+from paasta_tools.utils import INSTANCE_TYPES
 from paasta_tools.utils import load_system_paasta_config
 
 log = logging.getLogger(__name__)
 
+INSTANCE_TYPE_TO_K8S_NAMESPACE = {
+    "marathon": "paasta",
+    "adhoc": "paasta",
+    "kubernetes": "paasta",
+    "tron": "tron",
+    "flink": "paasta",
+    "cassandracluster": "paasta-cassandraclusters",
+    "kafkacluster": "paasta",
+    "nrtsearchservice": "paasta",
+}
```

Comment:
Could you please double-check that namespaces are correct for all instance types.
Specifically, why we use `paasta` namespace for `flink`/`kafkacluster`/`nrtsearchservice` instance type and not `paasta-flinks`/`paasta-kafkaclusters`/`paasta-nrtsearchservices` namespaces?